### ISSUE TITLE
MIT licensing issue

### ISSUE THREAD

**Issue by lae** [respawner/ansible-role-netbox#9]
Hi Guilliaume,

I notice that in 7c400dd5fab4070b0e0c02597d72a94449730193 there are significant portions of code that appear to be copied from [my project](https://github.com/lae/ansible-role-netbox). I'm glad to have been a good resource, especially since this role is more inline with how the upstream NetBox documentation details installation procedures. Thanks!

This was probably an oversight, but I noticed that it was missing a copy of the license from the original project, which is a requirement for redistribution/modification:

https://github.com/lae/ansible-role-netbox/blob/master/LICENSE

Since this repository is originally under GPL, there are two steps that need to be taken to resolve this:

1. A copy of the original MIT license needs to be present in this repository (probably best that it's in the parent directory since the code in concern doesn't appear to be isolated to a single subdirectory), and it needs to be clear to users that code in this repository, starting from that commit, falls under both licenses.
2. A reference to the original license should be present in the header of any files with significant (i.e. non-trivial) portions of code from the original repository in order to distinguish it from the GPL licensed code - e.g. the first line from the `LICENSE` linked above would suffice as a comment. You should also include a line referencing the GPL if there are significant portions that *don't* fall under the MIT license. I believe the following files are candidates:
  a. `tasks/install_packages.yml`
  b. `tasks/setup_netbox.yml`
  c. `templates/configuration.py.j2`
  d. `vars/main.yml`

Basically, this doesn't result in any change of permissions granted for this repository (as GPL is more or less a superset of MIT), but rather a change of attribution.

If you want to make those changes, that'd be great. Otherwise, I can submit a pull request myself. Also, sorry if I sound heavy-handed, I'm trying my best not to, but I poured a lot of time into this project and believe integrity is pretty important in these sorts of situations.

**Comment by respawner** [comment]
Hi @lae 

Indeed your code was of a great help.

Would you mind to open a PR submitting your changes to be compliant with the MIT?
In this way, I'll be glad to merge it and include what you think is best.

Non offense taken, there were no intent to spoil your work whatsoever and I'll be glad to honor it, the way it needs to be :)
